### PR TITLE
Story1

### PR DESCRIPTION
**Pull request by MotNosniktaw**

### CODE REVIEW

**Comment by RachmannJoubert** [comment]
It all looks good to me.

File: functions.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * creates an instance of PDO that access Guitars database
+ *
+ * @return PDO
+ */
+function getDB()
```
Comment (sjr-mayden) [inline functions.php]: Not OOP so brackets don't need to go on a new line

File: functions.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+/**
+ * Takes multidimensional array to be output as a string
+ * Outputs string containing Guitar Item values separated into separate divs and "row" container divs for respective Guitar Items
+ *
+ * @param array $guitarsArray
+ * @return string
+ */
+function displayGuitars(array $guitarsArray):string {
+    $result = '';
+        foreach ($guitarsArray as $guitar) {
```
Comment (sjr-mayden) [inline functions.php]: Exdent these lines

File: functions.php [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            $x = array_search($guitarsArray, $guitar);
+            if (isset($guitarsArray[$x]['id'])
+                && isset($guitarsArray[$x]['fileLocation'])
+                && isset($guitarsArray[$x]['brand'])
+                && isset($guitarsArray[$x]['model'])
+                && isset($guitarsArray[$x]['year'])
+                && isset($guitarsArray[$x]['type'])
+                && isset($guitarsArray[$x]['country'])
+                && isset($guitarsArray[$x]['value'])
+                && isset($guitarsArray[$x]['dateAcquired'])) {
```
Comment (sjr-mayden) [inline functions.php]: `) {` onto a new line

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Pull request merged**